### PR TITLE
Add Coin Wallet to the list of wallets

### PR DESCRIPTION
**Pull request by mahnunchik**
Could you please add Coin Wallet to the list of wallets?

Coin Wallet is a non-custodial multicurrency crypto wallet. 

It's an open-source wallet, and the code is available here: https://github.com/CoinSpace/CoinSpace

Coin Wallet is available as a web wallet, a mobile wallet (iOS & Android), and desktop apps (Windows, Linux & MacOS).

### CODE REVIEW

**Comment by mahnunchik** [comment]
@imaginaryusername I've resolved merge conflict.

**Comment by nikashitsa** [comment]
Well 100k+ android users know it. https://play.google.com/store/apps/details?id=com.coinspace.app 

image

Some other android wallets from bch.info list:
[Flowee Pay](https://flowee.org/products/pay) 100+ android users
[Paytaca](https://paytaca.com/) 5k+ android users
[Electron Cash](https://electroncash.org/) 10k+ android users
[Zapit Wallet](https://zapit.io/) 5k+ android users

Come on, Coin Wallet has over 100k android users.

**Review by merc1er — APPROVED** [review]
I have tried "Coin Wallet" a few times. It's actually a web wallet that is also embedded as an app.

Personally, I think that besides looking and feeling really like a website on mobile, it works quite well.

I tried their [web wallet](https://coin.space/wallet/) for the first time in a while, and was surprised to see the interface and user experience was much better than the iOS app.

Here is the comparison:
mobile
desktop

It also looks like the Desktop app (at least on macOS) is the mobile app ported over to Mac - which looks very "android"-like.

**In conclusion, I'm in favor of adding it, especially to the "Browser wallets" section.** I think it's also OK to add it to the other sections @imaginaryusername.

**Comment by merc1er** [comment]
We could add at some point a note in the third-party services pages (wallet, explorer, exchange pages) that we do not operate any of them and we cannot guarantee that they are safe.

But that's out of scope for this PR.

**Comment by imaginaryusername** [comment]
@mahnunchik @nikashitsa if one of you guys can resolve the conflict I'll merge it

**Comment by nikashitsa** [comment]
@merc1er @imaginaryusername done, thanks!
